caliban/docker.py

```diff
@@ -391,9 +391,17 @@ def _notebook_entries(lab: bool = False, version: Optional[str] = None) -> str:
 
   library = "jupyterlab" if lab else "jupyter"
 
-  return """
+  if not dlvm:
+    return """
 RUN pip install {}{}
 """.format(library, version_suffix)
+  else:
+    return """
+RUN /opt/conda/bin/pip install \
```

Comment:
does this only work on the deep learning VMs? and does it work on ALL the deep learning VMs?

If it works without the dlvms, maybe it needs its own flag.